datamodel/referencedata.d/shipmentinformationtypecodes.csv

```diff
@@ -0,0 +1,11 @@
+shipment_information_type_code,shipment_information_type_name
+TRD,Transport document
+BKG,Booking
+SHI,Shipping Instructions
+SRM,Shipment Release Message
+OOG,Out-of-Gauge declaration
+DGD,Dangerous Goods Declaration
+VGM,Verified Gross Mass
```

Comment:
It was agreed in P1 that VGM is to be removed from ShipmentEvent

Reply:
...it was removed because it requires a link to an equipment (container number) which the ShipmentEvent does not cater for